### PR TITLE
omhttpfs: rewrite documentation

### PR DESCRIPTION
**Pull request by PascalWithopf**
Module documentation was rewritten in the new style.

### CODE REVIEW

**Comment by deoren** [comment]
@PascalWithopf Not an objection, just a musing of sorts:

Is there value in listing the default value for a parameter in the description text for a parameter if it is already included in the table just above it? 

* Pro: help make the default more obvious
* Con: should a default value be changed we will have two places to update it

**Comment by PascalWithopf** [comment]
@deoren I think this should be judged on an individual basis. Sometimes it is unnecessary to do so and then I would take it out of the description, but there are also cases were the default is explained or further information is given, then I would leave it in there.

File: source/configuration/modules/omhttpfs.rst [hunk 1]
```diff
@@ -1,58 +1,149 @@
...
+^^^^
+
+.. csv-table::
+   :header: "type", "default", "mandatory", "|FmtObsoleteName| directive"
+   :widths: auto
+   :class: parameter-table
+
+   "word", "127.0.0.1", "no", "none"
+
+HttpFS server host. Default: *127.0.0.1*
```
Comment (deoren) [inline source/configuration/modules/omhttpfs.rst]: @PascalWithopf Here is an example which prompted my question.

**Comment by deoren** [comment]
> @PascalWithopf: @deoren I think this should be judged on an individual basis. Sometimes it is unnecessary to do so and then I would take it out of the description, but there are also cases were the default is explained or further information is given, then I would leave it in there.

Acknowledged. That makes sense. I picked out one example from the doc rewrite to accompany my question (I should have done that to start with).

**Comment by PascalWithopf** [comment]
@deoren I looked over the parameters and deleted all unnecessary repetitions.

**Comment by deoren** [comment]
@PascalWithopf Thanks!

**Pull request merged**